lms/templates/courseware/course_about.html

```diff
@@ -40,9 +40,14 @@
           location.href = xhr.responseText;
         }
       } else if (xhr.status == 403) {
-        $('#register_error').text(
-            (xhr.responseText ? xhr.responseText : "${_("An error has occurred. Please ensure that you are logged in to enroll.") | n, js_escaped_string}")
-        ).css("display", "block");
+        %if settings.REDIRECT_UNAUTHENTICATED_USER_TO_LOGIN_ON_ENROLL:
+          ## xss-lint: disable=mako-invalid-js-filter
```

Comment:
This is troubling. This is because of the `next=${request.path | u}` bit? It's been a while since I mucked with mako templates, so I'm not sure on what our xss-linter wants from you here, but disabling an xss-lint warning feels bad to me.

Reply:
If it is unavoidable and we are doing the correct thing here, at least a code comment for why we are fine here might be good.

Reply:
@mikix, I've rewritten this in a way that we don't need to suppress the linter here, by leveraging `encodeURIComponent` in JS side.